src/webapp/reports/autogenerated-forms/DataEntryItem.tsx

```diff
@@ -31,14 +31,17 @@ export interface DataEntryItemProps {
     columnDataElements?: DataElement[];
     cocId: string;
     rows?: Row[];
+    lockException?: boolean; // If true, the input will not be disabled if expired
```

Comment:
Not sure if I'm missing something but if the period is expired and we force an input to be enabled user can't enter data due to an exception from the API, right? 

Not wrong, just to be sure I understand how this works.

Reply:
yes that is the usual behaviour. I think it would be better I rewrite the comment to explain the behaviour of `lockException` here. In the TUB dataset, the previous periods are not locked by the API so this will work.

I think a better comment would be:
`// If true, the input will be not disabled when the period is not the same as the data form period`

how about that @eperedo?